subscriptions-api-openapi.yaml

```diff
@@ -110,6 +116,7 @@ paths:
         "404":
           description: Subscription not found
     options:
+      operationId: getSubscriptionFeatures
```

Comment:
You have "POST" below on line 128 (or 135) - I don't think we define "POST" for subscriptions/id URL

Reply:
You are correct. Good catch.